### ISSUE TITLE
use fused batch normalization

### ISSUE THREAD

**Issue by bodokaiser** [bodokaiser/mrtoct-tensorflow#11]
Should improve performance and be simple to enable just by passing `fuse=True` to [`tf.layers.BatchNormalization`][1].

[1]: https://www.tensorflow.org/versions/master/api_docs/python/tf/layers/BatchNormalization
